plugins/chunter-impl/src/components/Message.svelte

```diff
@@ -59,23 +57,31 @@
     loc.path.length = 4
     navigate(loc)
   }
+
+  function isToday (value: Date | Timestamp): boolean {
+    const today = new Date(new Date().getFullYear(), new Date().getMonth(), new Date().getDate()).getTime()
+    const date = new Date(value).getTime()
+    return date - today > 0
+  }
 </script>
 
 {#await getUser(message.modifiedBy) then user}
   <div class="message-container" on:click={onClick}>
-    <div class="avatar"><img src={user?.avatar ?? ''} alt="Avatar" /></div>
+    <div class="avatar"><img src={user?.avatar ?? ''} alt={user?.name} /></div>
```

Comment:
Probable we need some default account image in case there is no user one specified.